tests/integration/ambient/waypoint_test.go

```diff
@@ -381,20 +381,26 @@ func TestWaypointDNS(t *testing.T) {
 				if src.Config().HasSidecar() {
 					t.Skip("TODO: sidecars don't properly handle use-waypoint")
 				}
+				address := "240.240.240.239"
+				if _, v6 := getSupportedIPFamilies(t); v6 {
```

Comment:
I think we can unconditionally put both. Did you see issues?

Reply:
I didn't try using both.

Reply:
I think that putting both addresses can't work, because `Address` is used directly to [build](https://github.com/istio/istio/blob/08dcbe8456f02526c4fdc36356e71e6dc9e210e9/pkg/test/framework/components/echo/common/call.go#L210) the URL, so it won't work with multiple addresses. I tried putting both to be sure that Go can't handle it and it was failing.